services/QuillLMS/client/app/bundles/Shared/components/shared/tooltip.tsx

```diff
@@ -93,7 +93,7 @@ class Tooltip extends React.Component<TooltipProps, { clickedFromMobile: boolean
     const { tooltipVisible, } = this.state
     const tabIndex = isTabbable ? 0 : null;
 
-    const triggerClass = `quill-tooltip-trigger ${tooltipVisible ? 'active' : ''}`
+    const triggerClass = `quill-tooltip-trigger  ${tooltipVisible ? 'active' : ''}`
```

Comment:
Looks like an extra space snuck in here that's not necessary?

Reply:
ahh yeah, please disregard! I was testing something and must have accidentally left a space